src/imp/windows.rs

```diff
@@ -0,0 +1,166 @@
+// Distributed under the OSI-approved BSD 3-Clause License.
+// See accompanying LICENSE file for details.
+
+// Implemented by JR Andreassen jr at prtgconsultants com
+// references: 
+//   https://github.com/denisbrodbeck/machineid 
+//   https://github.com/jojonv/rust-win32error/blob/master/src/lib.rs
+
+use winreg::enums::{
+    HKEY_LOCAL_MACHINE, KEY_READ,
+};
+use log::debug;
+use thiserror::Error;
+use uuid::{self, Uuid};
+use std::io::{self};
+
+use rust_win32error::Win32Error;
```

Comment:
```suggestion
use std::io;

use log::debug;
use rust_win32error::Win32Error;
use thiserror::Error;
use uuid::{self, Uuid};
use winreg::enums::{
    HKEY_LOCAL_MACHINE, KEY_READ,
};
```

And then however `rustfmt` wants that to look.

Reply:
ran "cargo rustfmt" on the project...